app/assets/stylesheets/utilities/_overrides.scss

```diff
@@ -75,3 +75,15 @@
   }
 
 }
+
+.govuk-summary-list__key,
+.govuk-summary-list__value,
+.govuk-summary-list__actions {
+  border-bottom: 0;
+
+  // only apply border to the last row of the summary list
+  .govuk-summary-list .govuk-summary-list__row:last-child & {
```

Comment:
I wonder if this is an oversight in the gem. This component was copied from content-publisher, which does not display a line between each summary item, so perhaps rather than having an override here, this styling should be added to the gem?

Reply:
Ah, looks like the summary_list component in [govuk_publishing_components inherits from govuk-frontend](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_summary-list.scss#L1) (i.e. Design System). In Design System, it [_does_ have the horizontal lines in between rows](https://design-system.service.gov.uk/components/summary-list/) (although there is the [option to override it with `govuk-summary-list--no-border`](https://design-system.service.gov.uk/components/summary-list/#summary-list-without-borders).

Unfortunately there's no way to pass that class to the component, so I'm still going to have to make a PR to govuk_publishing_components 😅

Reply:
Have raised https://github.com/alphagov/govuk_publishing_components/pull/1073.

Reply:
Merged PR. Now blocked on getting the gem bumped & published.

Once that's done, can pull in latest gem, remove the override commit, take new screenshots for the PR description and get this merged 🎉